src/Mod/PartDesign/Gui/Command.cpp

```diff
@@ -1541,7 +1541,7 @@ bool CmdPartDesignSubtractiveHelix::isActive()
 //===========================================================================
 
 bool dressupGetSelected(Gui::Command* cmd, const std::string& which,
-        Gui::SelectionObject &selected, bool &useAllEdges)
+        Gui::SelectionObject &selected, bool &useAllEdges, bool& noSelection)
```

Comment:
I'm a bit concerned about "boolean blindness" here -- we've got a function that effectively returns _three_ booleans now. Consider filing this away as a potential spot for a future refactoring.